packages/compiler/src/server/serverlib.ts

```diff
@@ -860,14 +861,34 @@ export function createServer(host: ServerHost): Server {
       const tspDiag = currentDiagnosticIndex.get(vsDiag.data?.id);
       if (tspDiag === undefined || tspDiag.codefixes === undefined) continue;
 
+      if (
+        vsDiag.codeDescription?.href &&
+        tspDiag.codefixes?.findIndex((x) => x.id === "show-linter-rule-doc-url") === -1
+      ) {
+        mutate(tspDiag.codefixes).push(
```

Comment:
This will still mutate the program . This most likely needs to be done later right when creating the vs code actions